src/network/networkd-network-gperf.gperf

```diff
@@ -27,291 +27,302 @@ struct ConfigPerfItem;
+DHCPv6.UseNTP,                               config_parse_bool,                                        0,                             offsetof(Network, dhcp6_use_ntp)
+DHCPv6.RapidCommit,                          config_parse_bool,                                        0,                             offsetof(Network, rapid_commit)
+DHCPv6.ForceDHCPv6PDOtherInformation,        config_parse_bool,                                        0,                             offsetof(Network, dhcp6_force_pd_other_information)
+DHCPv6.PrefixDelegationHint,                 config_parse_dhcp6_pd_hint,                               0,                             0
+IPv6AcceptRA.UseAutonomousPrefix,            config_parse_bool,                                        0,                             offsetof(Network, ipv6_accept_ra_use_autonomous_prefix)
+IPv6AcceptRA.UseOnLinkPrefix,                config_parse_bool,                                        0,                             offsetof(Network, ipv6_accept_ra_use_onlink_prefix)
+IPv6AcceptRA.UseDNS,                         config_parse_bool,                                        0,                             offsetof(Network, ipv6_accept_ra_use_dns)
+IPv6AcceptRA.UseDomains,                     config_parse_dhcp_use_domains,                            0,                             offsetof(Network, ipv6_accept_ra_use_domains)
+IPv6AcceptRA.RouteTable,                     config_parse_section_route_table,                         0,                             0
+IPv6AcceptRA.BlackList,                      config_parse_ndisc_black_listed_prefix,                   0,                             0
+DHCPServer.MaxLeaseTimeSec,                  config_parse_sec,                                         0,                             offsetof(Network, dhcp_server_max_lease_time_usec)
+DHCPServer.DefaultLeaseTimeSec,              config_parse_sec,                                         0,                             offsetof(Network, dhcp_server_default_lease_time_usec)
+DHCPServer.EmitDNS,                          config_parse_bool,                                        0,                             offsetof(Network, dhcp_server_emit_dns)
+DHCPServer.DNS,                              config_parse_dhcp_server_dns,                             0,                             0
+DHCPServer.EmitNTP,                          config_parse_bool,                                        0,                             offsetof(Network, dhcp_server_emit_ntp)
+DHCPServer.NTP,                              config_parse_dhcp_server_ntp,                             0,                             0
+DHCPServer.EmitSIP,                          config_parse_bool,                                        0,                             offsetof(Network, dhcp_server_emit_sip)
+DHCPServer.SIP,                              config_parse_dhcp_server
```

Comment:
@yuwata why you changed this to `QDISC_KIND_NETEM` .  Previously it was not associated with  anything.

Reply:
This is a trick. In v244, we have only netem support, So, [TrafficControlQueueingDiscipline] section can safely treat as a [NetworkEmulattor] section.

Reply:
I not able to replicate this behaviour. If I just want to add the following now it thinks 

```
[TrafficControlQueueingDiscipline]
Parent=ingress
```

Now it defaults to `qdisc netem ffff: dev dummy99 parent ffff:fff1 limit 1000` and it should not associate with anything actually. 

I cant replicate this ```qdisc ingress ffff: dev dummy99 parent ffff:fff1```

Reply:
Oh...

Reply:
we need to fix it.

Reply:
#14784. PTAL.